cli/lib/util.js

```diff
@@ -30,31 +30,11 @@ const getosAsync = Promise.promisify(getos)
 
 /**
  * Returns SHA512 of a file
- *
- * Implementation lifted from https://github.com/sindresorhus/hasha
- * but without bringing that dependency (since hasha is Node v8+)
  */
 const getFileChecksum = (filename) => {
   la(is.unemptyString(filename), 'expected filename', filename)
 
-  const hashStream = () => {
-    const s = crypto.createHash('sha512')
-
-    s.setEncoding('hex')
-
-    return s
-  }
-
-  return new Promise((resolve, reject) => {
-    const stream = fs.createReadStream(filename)
-
-    stream.on('error', reject)
-    .pipe(hashStream())
-    .on('error', reject)
-    .on('finish', function () {
-      resolve(this.read())
-    })
-  })
+  return hasha.fromFile(filename, { algorithm: 'sha512' })
```

Comment:
checks out to me https://github.com/sindresorhus/hasha/blob/v5.2.2/index.js#L120